packages/terra-breakpoints/src/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$terra-breakpoints: (
+  tiny: 0,
+  small: 544px,
+  medium: 768px,
+  large: 992px,
+  huge: 1216px,
+  enormous: 1440px
+);
```

Comment:
Do we still want to keep this file in sass. Are consumers using it in their custom scss?